nbgrader/exchange/ngshare/fetch_assignment.py

```diff
@@ -43,21 +42,13 @@ def init_dest(self):
             self.fail('You already have a copy of the assignment in this directory: {}'.format(root))
 
     def copy_files(self):
-        try:
-            params = {'user': self.username}
-            response = requests.get(self.src_path, params=params)
-        except:
-            self.log.warn('An error occurred while trying to fetch {}'.format(self.coursedir.assignment_id))
-
-        if response.status_code != requests.codes.ok:
-            self.log.warn('An error occurred while trying to fetch {}'.format(self.coursedir.assignment_id))
-            self.log.warn('Status Code: {}'.format(response.status_code))
-        elif not response.json()['success']:
-            self.log.warn('Failed to fetch assignment. Reason: {}'.format(response.json()['message']))
+        response = self.ngshare_api_get(self.src_path)
```

Comment:
Do we need to pass in `params = {'user': self.username}` to the get request?

Reply:
Unless you're using vngshare, no. I'll talk to @lxylxy123456 about somehow merging vngshare and ngshare so it can support both modes of operation, maybe.